README.md

```diff
@@ -57,7 +57,7 @@ To learn how to use restic backup client with REST backend, please consult [rest
           --log string           log HTTP requests in the combined log format
           --max-size int         the maximum size of the repository in bytes
           --no-auth              disable .htpasswd authentication
-          --path string          data directory (default "/tmp/restic")
+          --path string          data directory
```

Comment:
Does the rest-server print at which location is expects the data directory?

Reply:
Yes. However, now the path is platform-dependent. I deleted it as it might confuse users looking for default paths

Reply:
At least mention that the default is to store it in a temporary directory where you will lose it.

Reply:
I've changed a bit the description below - I added:

**If `rest-server` is launched using the default path, all backups will be lost**.

See: https://github.com/restic/rest-server/pull/158/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R76